src/VisualStudio/Core/Def/Implementation/Snippets/AbstractSnippetExpansionClient.cs

```diff
@@ -393,7 +452,317 @@ public virtual bool TryInsertExpansion(int startPositionInSubjectBuffer, int end
                 iEndIndex = endIndex
             };
 
-            return expansion.InsertExpansion(textSpan, textSpan, this, LanguageServiceGuid, out ExpansionSession) == VSConstants.S_OK;
+            if (expansion.InsertExpansion(textSpan, textSpan, this, LanguageServiceGuid, out _state._expansionSession) == VSConstants.S_OK)
+            {
+                // This expansion is not derived from a symbol
+                _state.ClearSymbolInformation();
```

Comment:
trying to determine the relationship between ClearSymbolInformation and `.Clear(forceClearSymbolInformation`.  not totally evident.

Reply:
⏱ Updating this to hopefully be more clear. There are limits considering the APIs.